configure.cmd

```diff
@@ -47,10 +47,11 @@ if defined ROS_ARCH (
 
 ) else if defined VCINSTALLDIR (
     REM VS command prompt does not put this in environment vars
-    cl 2>&1 | find "x86" > NUL && set ARCH=i386
-    cl 2>&1 | find "x64" > NUL && set ARCH=amd64
-    cl 2>&1 | find "ARM" > NUL && set ARCH=arm
-    cl 2>&1 | find "ARM64" > NUL && set ARCH=arm64
+
+    REM Set System Architecture Directly
+    set ARCH=%processor_architecture: =%
```

Comment:
Why not just setting `ARCH` to `PROCESSOR_ARCHITECTURE` like:
`set ARCH=%PROCESSOR_ARCHITECTURE%` ?